PostmanCollections/VTEX - Logistics API.json

```diff
@@ -4933,12 +4933,12 @@
           }
         },
         {
-          "id": "efccf715-4643-4fbf-8e64-88588c17b547",
+          "id": "f31a9f5b-eb16-489b-8168-64303ff4adb4",
           "name": "Update inventory by SKU and warehouse",
           "request": {
             "name": "Update inventory by SKU and warehouse",
             "description": {
-              "content": "Updates inventory for a given SKU and warehouse.",
+              "content": "Updates the [inventory](https://help.vtex.com/tutorial/inventory-management--tutorials_139) of a SKU unit in a given [warehouse](https://help.vtex.com/en/tutorial/warehouse--6oIxvsVDTtGpO7y6zwhGpb). This endpoint allows you to do the following:\r\n\r\n- **Define a quantity:** You will have a specific number of SKU units, and it will decrease whenever a unit is sold. The store can eventually run out of stock.\r\n\r\n- **Set an infinity inventory:** You will always have the SKU unit from the informed warehouse available for sales. The store cannot run out of stock.\r\n\r\nSuccessful requests receive a `200 OK` with `true` in the response body.",
```

Comment:
```suggestion
              "content": "Updates the [inventory](https://help.vtex.com/tutorial/inventory-management--tutorials_139) of an SKU unit in a given [warehouse](https://help.vtex.com/en/tutorial/warehouse--6oIxvsVDTtGpO7y6zwhGpb). This endpoint allows you to do the following:\r\n\r\n- **Define a quantity:** You'll indicate a specific number of SKU units, and it will decrease whenever a unit is sold. The store can eventually run out of stock.\r\n\r\n- **Set an infinite inventory:** You will always have the SKU unit from the informed warehouse available for sales. The store cannot run out of stock.\r\n\r\nSuccessful requests receive a `200 OK` with `true` in the response body.",
```

Reply:
- Troquei o "You **will have** a specific number" por "**You'll indicate** a specific number" pra ter mais agência de que o dev que insere o número desejado, ao invés de só "ter u m número que cai do ceu" hehe. 
- **An SKU**, ao invés de "a SKU".
- **Infinite** inventory é mais correto que _infinity_ e fica padronizado com a doc do [Help](https://help.vtex.com/en/tutorial/managing-stock-items--tutorials_139#inventory-information) e contexto de [Sales App](https://help.vtex.com/en/tutorial/enabling-local-stock-sale-in-vtex-sales-app--54eQN4rOH5yBYPGG2w8v9q)